gazebo_dev/cmake/gazebo_dev-extras.cmake

```diff
@@ -3,11 +3,40 @@ find_package(gazebo REQUIRED)
 
 message(STATUS "Gazebo version: ${GAZEBO_VERSION}")
 
+# Workaround for https://github.com/ros-simulation/gazebo_ros_pkgs/issues/1372
+# Real fix should go into Gazebo11 https://github.com/osrf/gazebo/issues/3008
+# On Ubuntu Jammy Gazebo11 is used directly from Ubuntu, before removing this
+# please be sure that the package is updated there.
+pkg_check_modules(TBB tbb)
+set (TBB_PKG_CONFIG "tbb")
```

Comment:
style nitpick: I would remove the extra space before each `(` in the added code to be consistent with the rest of the file.

Reply:
done in [15e2669](https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1373/commits/15e266905056b66f282a8eba793720248c8804df)